src/azure-cli/azure/cli/command_modules/aro/_params.py

```diff
@@ -64,9 +64,11 @@ def load_arguments(self, _):
         c.argument('worker_vm_size',
                    help='Size of worker VMs.')
         c.argument('worker_vm_disk_size_gb',
+                   type=int,
```

Comment:
there are corresponding diffs which need to be fixed up in _validators.py - please check